Source/ACE/Managers/LandblockManager.cs

```diff
@@ -87,6 +86,8 @@ private static Landblock GetLandblock(LandblockId landblockId, bool propogate)
                     {
                         // load up this landblock
                         Landblock block = new Landblock(landblockId);
+                        block.Load();

```

Comment:
this was just the start of loading and unloading the landblock.. It can be removed.. it has no bearing on this diag tool